src/components/evaluation/Chart.tsx

```diff
@@ -122,27 +102,23 @@ function Chart({
         if (ChartComponent) {
           return (
             <SizeMe refreshRate={250}>
-              {({ size }): React.ReactElement =>
-                (showQuestionLayout && (
-                  <ChartComponent
-                    brush={sessionStatus !== SESSION_STATUS.RUNNING}
-                    data={data}
-                    instanceId={instanceId}
-                    isColored={questionType !== 'FREE_RANGE'}
-                    isPublic={isPublic}
-                    isSolutionShown={showSolution}
-                    numBins={numBins}
-                    questionType={questionType}
-                    restrictions={restrictions}
-                    sessionId={sessionId}
-                    size={size}
-                    statistics={statistics}
-                    totalResponses={totalResponses}
-                  />
-                )) ||
-                (showConfusionTS && <ConfusionBarometerChart confusionTS={confusionTS} />) ||
```

Comment:
Basically pulled this to a higher-level, allowing me to remove all the passed props that are only required for confusion/feedback.